pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -184,24 +187,6 @@ func NewMultiCIDRRangeAllocator(
 		klog.V(0).Info("No Secondary Service CIDR provided. Skipping filtering out secondary service addresses.")
 	}
 
-	if nodeList != nil {
-		for _, node := range nodeList.Items {
-			if len(node.Spec.PodCIDRs) == 0 {
-				klog.V(4).Infof("Node %v has no CIDR, ignoring", node.Name)
-				continue
-			}
-			klog.V(0).Infof("Node %v has CIDR %s, occupying it in CIDR map", node.Name, node.Spec.PodCIDRs)
-			if err := ra.occupyCIDRs(&node); err != nil {
-				// This will happen if:
-				// 1. We find garbage in the podCIDRs field. Retrying is useless.
-				// 2. CIDR out of range: This means ClusterCIDR is not yet created
-				// This error will keep crashing controller-manager until the
-				// appropriate ClusterCIDR has been created
-				return nil, err
-			}
-		}
-	}
-
```

Comment:
this happens once the infomrer is synced, it gets a List() from the apiserver and inserts all items in the queue